modules/oadp-backing-up-applications.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing-up-and-restoring-applications.adoc
+
+[id="oadp-backing-up-applications_{context}"]
+= Backing up applications
+
+You back up applications by creating a `Backup` custom resource (CR).
+
+.Prerequisites
+
+* You must have the Data Protection Application installed.
```

Comment:
It might be worth noting that you must also have created a DPA CR manifest that is successful?

Reply:
If the DPA CR is not successful, the Data Protect Application would not be considered "installed". I will think about this.